QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
-Quick start, because I can't be arsed writing a full one ;)
+**Hello!**
 
-SQL setup
----------
 
-1. Create a user and database in your mysql server for the bot to use
-2. Populate the database using the .sql script
+Steps you need to do (or packages you should have) before downloading the script:
+1. Install all necessary packages for eggdrop to make use of mysql (also packages which you need to have are included).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: tcl, tcl-dev, tcllib, tcl-tls, zlibc, zlib1g, tcl-trf, mysqltcl
+2. Install mysql-server and phpmyadmin (phpmyadmin is not really needed, but it helps to manage databases).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: mysql-server, mysql-client, phpmyadmin
+      <br>
+      For phpmyadmin you need the following packages: apache2, php <- install those first. :)
+3. Install eggdrop. Check out here: https://www.eggheads.org/downloads
 
-Note: Give the SQL user minimum permissions, restrict it to just the host the bot runs on, etc.
```

Comment:
I'd prefer to keep the line about minimum permissions etc in (in some form at least, if you want to reword it).

Reply:
left a note like that in the beginning.